src/components/page/Nightingale/Nightingale.svelte

```diff
@@ -436,7 +448,10 @@
     width: 100%;
     max-width: 600px;
     &.apartment {
-      background: url("https://res.cloudinary.com/future-super/image/upload/f_auto,q_auto/v1639300170/nightingale-apartment.png");
+      background: url("https://res.cloudinary.com/future-super/image/upload/v1639300170/nightingale-kitchen.png");
+      background-repeat: no-repeat;
+      background-size: cover;
+      background-position: center;
```

Comment:
Needed to specify no-repeat again + swapped the picture for what's on Figma.

before:

<img src="https://github.com/futuresuper/fs-website-2021/assets/125414123/c5e3f694-1109-4776-8c92-c8aec3bfd7cc" width=300 />

after:

<img src="https://github.com/futuresuper/fs-website-2021/assets/125414123/f1082269-24a0-4055-9f04-33dc016413be" width=300 />